<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VIDHYARAJENDRAN" failures="1" tests="3" name="ErrorValidationTest" time="8.254" errors="0" timestamp="2024-05-22T23:32:40 IST">
  <testcase classname="com.tests.ErrorValidationTest" name="LoginErrorValidation" time="0.331">
    <skipped/>
  </testcase> <!-- LoginErrorValidation -->
  <testcase classname="com.tests.ErrorValidationTest" name="LoginErrorValidation" time="0.307">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.tests.ErrorValidationTest.LoginErrorValidation(ErrorValidationTest.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginErrorValidation -->
  <testcase classname="com.tests.ErrorValidationTest" name="ProdErrorValidation" time="0.647"/>
</testsuite> <!-- ErrorValidationTest -->
